src/site/content/en/blog/optimize-ttfb/index.md

```diff
@@ -0,0 +1,229 @@
+---
+title: Optimize Time to First Byte
+subhead: |
+  Learn how to optimize for the Time to First Byte metric.
+authors:
+  - jlwagner
+date: 2023-01-18
+hero: image/jL3OLOhcWUQDnR4XjewLBx4e3PC3/IuvYBfNBVKH7qU3872qg.jpg
+thumbnail: image/jL3OLOhcWUQDnR4XjewLBx4e3PC3/OBITO4G8MUJMILC7IFE0.jpg
+alt: A picture of several racks of servers, with white text superimposed on it that reads "Optimize Time to First Byte".
+description: |
+  Learn how to optimize for the Time to First Byte metric.
+tags:
+  - blog
+  - performance
+  - web-vitals
+---
+
+[Time to First Byte (TTFB)](/ttfb/) is a foundational web performance metric that precedes every other meaningful user experience metric such as [First Contentful Paint (FCP)](/fcp/) and [Largest Contentful Paint (LCP)](/lcp/). This means that high TTFB values add time to the metrics that follow it.
+
+It's recommended that your server responds to navigation requests quickly enough so that the **75th percentile** of users experience an FCP [within the "good" threshold](/fcp/#what-is-a-good-fcp-score). As a rough guide, most sites should strive to have a TTFB of **0.8 seconds or less**.
+
+<figure>
+  <picture>
+    <source
+      srcset="{{ "image/W3z1f5ZkBJSgL1V1IfloTIctbIF3/ILJ1xKjzVisqOPPyHYVA.svg" | imgix }}"
+      media="(min-width: 640px)"
+      width="800"
+      height="200">
+    {%
+      Img
+        src="image/W3z1f5ZkBJSgL1V1IfloTIctbIF3/EcKicxW5ErYYhf8RvpeO.svg",
+        alt="Good TTFB values are 0.8 seconds or less, poor values are greater than 1.8 seconds, and anything in between needs improvement",
+        width="640",
+        height="480"
+    %}
+  </picture>
+</figure>
+
+{% Aside 'important' %}
+TTFB is not a Core Web Vitals metric, so it's not absolutely necessary that sites meet the "good" TTFB threshold, provided that it doesn't impede their ability to score well on the metrics that matter.
+
+Websites vary in how they deliver content. A low TTFB is crucial for getting markup out to the client as soon as possible. However, if a website delivers the initial markup quickly, but that markup then requires JavaScript to populate it with meaningful content—as is the the case with Single Page Applications (SPAs)—then achieving the lowest possible TTFB is especially important so that the client-rendering of markup can occur sooner.
+
+Conversely, a server-rendered site that does not require as much client-side work may have a slightly lower TTFB, but good FCP and LCP values. This is why the TTFB thresholds are a “rough guide”, and will need to be weighed against how your site delivers its core content.
+{% endAside %}
```

Comment:
Looks like you need to use `<p>` tags in an `{% Aside %}` to break up paragraphs:

```suggestion
{% Aside 'important' %}
<p>TTFB is not a Core Web Vitals metric, so it's not absolutely necessary that sites meet the "good" TTFB threshold, provided that it doesn't impede their ability to score well on the metrics that matter.<p>

<p>Websites vary in how they deliver content. A low TTFB is crucial for getting markup out to the client as soon as possible. However, if a website delivers the initial markup quickly, but that markup then requires JavaScript to populate it with meaningful content—as is the the case with Single Page Applications (SPAs)—then achieving the lowest possible TTFB is especially important so that the client-rendering of markup can occur sooner.<p>

<p>Conversely, a server-rendered site that does not require as much client-side work may have a slightly lower TTFB, but good FCP and LCP values. This is why the TTFB thresholds are a “rough guide”, and will need to be weighed against how your site delivers its core content.<p>
{% endAside %}
```